internal/invocation/invocation.go

```diff
@@ -152,14 +152,10 @@ func (i *Impl) CanRetry(err error) bool {
 		}
 	*/
 
-	var ioError *hzerrors.HazelcastIOError
-	var instanceNotActiveError *hzerrors.HazelcastInstanceNotActiveError
-	if errors.As(err, &ioError) || errors.As(err, &instanceNotActiveError) {
+	if errors.Is(err, hzerrors.ErrIO) || errors.Is(err, hzerrors.ErrHazelcastInstanceNotActive) {
```

Comment:
Will we add the TODO logic above in the stable release? I feel like this is important

Reply:
We don't yet have anything that requires invoke on member, so there is no invokeOnMember in the client yet.

Reply:
I see that we have PN Counter. Looking at the Java side, we use UUID based invocation there. https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/client/impl/proxy/ClientPNCounterProxy.java#L258

https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/client/impl/proxy/ClientPNCounterProxy.java#L300

What we do in those cases in Go client?

Reply:
My bad, it seems we have something that corresponds to invokeMember after all, but no invokeMember function. Let me see how I can resolve that TODO.

Reply:
Implemented at: https://github.com/hazelcast/hazelcast-go-client/pull/586/commits/e02caad44506632b0bb7c982369b3bd9cec4f9d4